_posts/0000-01-02-jhnrustan1978.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)